src/lang/modelingWorkflows.ts

```diff
@@ -2,7 +2,7 @@
  * Modeling Workflows
  *
  * This module contains higher-level CAD operation workflows that
- * coordinate between different subsystems in the modeling app:
+ * coordinate between different subsystems in the Design Studio:
```

Comment:
Nitpick to ignore, but if code comments like this simply said "the app" we'd never have to change them again.